tests/test_e2e.py

```diff
@@ -200,6 +200,7 @@ def _build_app(cls,
                 http_client=http_client or MinimalHttpClient(),
                 enable_broker_on_windows=broker_available,
                 enable_broker_on_mac=broker_available,
+                enable_broker_on_linux=broker_available,
```

Comment:
**praise:** Thanks for finding this test file!

**suggestion:** If you haven't already, please also do `python -m unittest tests.test_e2e` and make sure we won't run into any issue. Note that you will need to setup an `.env` first, whose content is described at the beginning of this test_e2e.py file.